force-app/main/default/permissionsets/SA_Administrator.permissionset-meta.xml

```diff
@@ -211,16 +211,36 @@
         <field>Case.ParentId</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
+        <editable>false</editable>
+        <field>Case.Patient_Address_Template__c</field>
```

Comment:
@azfanpwc Mind committing this missing field? Or remove the xml block if the field isn't intended for release.